packages/babel-plugin-proposal-optional-chaining/src/index.js

```diff
@@ -72,12 +83,12 @@ export default declare((api, options) => {
 
           // Ensure call expressions have the proper `this`
           // `foo.bar()` has context `foo`.
-          if (isCall && t.isMemberExpression(chain)) {
```

Comment:
We'll still need this.

Reply:
Ah, I was wondering why the chain.object wasn't needed before. Do you think isMemberExpression is a bit too specific? Shouldn't we allow optional member expressions too?

Reply:
Any optional members will have been converted to regular members by https://github.com/babel/babel/blob/d4d6ec8f2b8ebfb32381a1e8dcadbadec3ae5736/packages/babel-plugin-proposal-optional-chaining/src/index.js#L41